foreign_cc/private/configure_script.bzl

```diff
@@ -38,8 +38,8 @@ def create_configure_script(
         root_path = "$$BUILD_TMPDIR$$"
         configure_path = "{}/{}".format(root_path, configure_command)
 
-    script.append("##export_var## MAKE {}".format(make_path))
     script.append("##enable_tracing##")
+    script.append("##export_var## MAKE {}".format(make_path))
```

Comment:
Why flip these?

Reply:
It was so that I could see what `MAKE` was being set to as the cmake build was complaining that it didn't have a command set for `CMAKE_MAKE_PROGRAM` which I'd assumed would pick up `MAKE` but it doesn't.
I can revert this change if you like?

Reply:
Yeah, I think this would show up in the environment dump of a failed build so if something isn't working we'd know what was used where. I know it's a small change but being able to keep the generally massive log size as small as possible is a win to me 😅

Reply:
Reverted